src/components/state-details.js

```diff
@@ -581,6 +578,23 @@ export default function StateDetailsPage ({ location, data }) {
                 </>
               )}
 
+              <div className="card">
+                <div className="card-body">
+                  <h5 class="card-title">
```

Comment:
This `h5` is following an `h2`, so it's skipping levels. Also, there's no content under this so I don't think a heading is a good fit. Just a `<p>` or `<span>` is fine.